bootcamp_module08/core/tests/test_student_11.py

```diff
@@ -19,6 +28,24 @@ def test_count_substring_repeated():
     assert expected_count == observed_count
 
 
+def test_count_substring_repeated_cases():
+    test_string = "AGcTAGCAGT"
+    test_substring = "agc"
+
+    expected_count = 2
+    observed_count = count_substring(test_string, test_substring)
+    assert expected_count == observed_count
+
+
+def test_count_substring_cases_cont():
```

Comment:
Not sure what `cont` stands for, this test could have more informative name.
What about `test_count_substring_caseless_insensitive_4`